ql/src/java/org/apache/hadoop/hive/ql/exec/tez/DagUtils.java

```diff
@@ -227,6 +228,14 @@ public Object process(Node nd, Stack<Node> stack, NodeProcessorCtx procCtx,
       return null;
     }
   }
+  public static String getQueryName(Configuration conf) {
```

Comment:
you haven't needed to change any reference? isn't this used at all? if so, I'm fine with removing it :)

Reply:
Yup, no usage that the ide could detect as well, will remove it :)